### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by NouemanKHAL**
General improvement of the documentation in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,41 +23,12 @@ Once it is available in your Cloud Foundry environment, configure your applicati
...
-#### Application Metadata collection
-
-> This feature requires both the Datadog Agent and the Datadog Cluster Agent to be installed. 
-See [Datadog Agent BOSH Release](https://github.com/DataDog/datadog-cluster-agent-boshrelease) and [Datadog Cluster Agent BOSH Release](https://github.com/DataDog/datadog-cluster-agent-boshrelease).
-
-You can enable the collection of your application metadata (labels and annotations) as tags in your application logs, traces and metrics by setting the environment variable `DD_ENABLE_CAPI_METADATA_COLLECTION` to `true`.
-
-__Note__: Enabling this feature might trigger a restart of the Datadog Agent when the application metadata are updated, depending on the `cloud_foundry_api.poll_interval` on the Datadog Cluster Agent. On average, it takes around 20 seconds to restart the agent.
+### General configuration of the Datadog Agent
+All the options supported by the Agent in the main `datadog.yaml` configuration file can also be set through environment variables as described in the [documentation of the Agent][5].
```
Comment (hestonhoffman) [inline README.md:29]:
```suggestion
All options supported by the Agent in the main `datadog.yaml` configuration file can also be set through environment variables as described in the [Agent documentation][5].
```

File: README.md [hunk 2]
```diff
@@ -66,7 +37,12 @@ env:
   DD_API_KEY: <DATADOG_API_KEY>
 ```
 
-#### Log collection
+### Instrument your application
```
Comment (hestonhoffman) [inline README.md:40]:
```suggestion
### Instrument your application

```

File: README.md [hunk 3]
```diff
@@ -108,12 +84,38 @@ env:
...
 ```
 
-#### General configuration of the Datadog Agent
-All the options supported by the Agent in the main `datadog.yaml` configuration file can also be set through environment variables as described in the [documentation of the Agent][5].
+### Unified Service Tagging
+
+> This feature requires the Datadog Cluster Agent to be installed. 
+See [Datadog Cluster Agent BOSH Release](https://github.com/DataDog/datadog-cluster-agent-boshrelease).
+
+Unified service tagging ties Datadog telemetry together through using three reserved tags: `env`, `service`, and `version`. In Cloud Foundry, they are set through the application labels/annotations and `DD_ENV`, `DD_SERVICE` and `DD_VERSION` environment variables, as shown in the example below:
```
Comment (hestonhoffman) [inline README.md:92]:
```suggestion
Unified service tagging ties Datadog telemetry together using three reserved tags: `env`, `service`, and `version`. In Cloud Foundry, they are set through the application labels/annotations and `DD_ENV`, `DD_SERVICE` and `DD_VERSION` environment variables, as shown in the example below:
```

File: README.md [hunk 4]
```diff
@@ -108,12 +84,38 @@ env:
...
+    DD_SERVICE: <SERVICE_NAME>
+    DD_VERSION: <VERSION>
+  metadata:
+    labels:
+      tags.datadoghq.com/env: <ENV_NAME>
+      tags.datadoghq.com/service: <SERVICE_NAME>
+      tags.datadoghq.com/version: <VERSION>
+```
+
+The `tags.datadoghq.com` prefix is part of the Agent Autodiscovery notation as described in [Basic Agent Autodiscovery Documentation](https://docs.datadoghq.com/getting_started/containers/autodiscovery).
```
Comment (hestonhoffman) [inline README.md:106]:
```suggestion
The `tags.datadoghq.com` prefix is part of the Agent Autodiscovery notation as described in [Basic Agent Autodiscovery documentation](https://docs.datadoghq.com/getting_started/containers/autodiscovery).
```

File: README.md [hunk 5]
```diff
@@ -108,12 +84,38 @@ env:
...
+
+You can find more information in the [Unified Service Tagging Documentation](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging).
```
Comment (hestonhoffman) [inline README.md:108]:
```suggestion
You can find more information in the [Unified Service Tagging documentation](https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging).
```

**Review by hestonhoffman — APPROVED** [review]
Some style updates, but LGTM!

**Review by sarah-witt — DISMISSED** [review]

**Review by sarah-witt — APPROVED** [review]

**Pull request merged**